src/NzbDrone.Core.Test/MediaFiles/MediaInfo/MediaInfoFormatterTests/FormatAudioCodecFixture.cs

```diff
@@ -0,0 +1,77 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Core.MediaFiles.MediaInfo;
+
+namespace NzbDrone.Core.Test.MediaFiles.MediaInfo.MediaInfoFormatterTests
+{
+    [TestFixture]
+    public class FormatAudioCodecFixture
+    {
+        [Test]
+        public void should_return_AC3_for_AC_3()
```

Comment:
I'd simply these tests like the parser tests, with `[TestCase(format, profile, result)]should_simplify_format`  and `[TestCase(format)]should_keep_format`.
That way it's easier to add new cases.

Same for video, of course.